code/packages/web/src/pages/SignupPage.css

```diff
@@ -1,149 +1,167 @@
-  }
-  
-  .signup-box form {
-    display: flex;
-    flex-direction: column;
-  }
-  
-  .signup-box label {
-    font-size: 1rem;
-    margin-bottom: 5px;
-    color: #333;
-  }
-  
-  .signup-box input {
-    padding: 10px;
-    font-size: 1rem;
-    margin-bottom: 20px;
-    border: 1px solid #ccc;
-    border-radius: 5px;
-  }
-  
-  .signup-btn {
-    padding: 15px;
-    background: linear-gradient(90deg, #0d3dff, #0037b3);
-    color: white;
-    border: none;
-    border-radius: 5px;
-    cursor: pointer;
-    font-size: 1rem;
-    transition: background 0.3s;
-  }
-  
-  .signup-btn:hover {
-    background: linear-gradient(90deg, #0a35e3, #002ea5);
-  }
-  
-  /* Social Login Buttons */
-  .social-login {
-    text-align: center;
-    margin-top: 20px;
-  }
-  
-  .social-login p {
-    margin-bottom: 10px;
-  }
-  
-  .social-icons button {
-    margin: 0 10px;
-    padding: 10px;
-    font-size: 1.5rem;
-    border: none;
-    border-radius: 50%;
-    cursor: pointer;
-  }
-  
-  .google-login {
-    background-color: #db4437;
-    color: white;
-  }
-  
-  .facebook-login {
-    background-color: #3b5998;
-    color: white;
-  }
-  
-  .github-login {
-    background-color: #333;
-    color: white;
-  }
-  
-  /* Footer Links */
-  .login-text {
-    text-align: center;
-    margin-top: 20px;
-  }
-  
-  .footer-links {
-    display: flex;
-    justify-content: space-between;
-    margin-top: 20px;
-    font-size: 0.9rem;
-  }
-  
-  .footer-links a {
-    color: #007bff;
-    text-decoration: none;
-  }
-  
\ No newline at end of file
+/* General page styling */
+.signup-page {
+  display: flex;
+  height: 100vh;
+  background: linear-gradient(135deg, #cfd9df, #e2ebf0);
+  align-items: center;
+  justify-content: center;
+}
+
+.left-section {
+  flex: 1;
+  text-align: center;
+  padding: 50px;
+  background: linear-gradient(to bottom, #e0e5f7, #cfd7f5);
+}
+
+.left-section h1 {
+  font-size: 3em;
+  font-weight: bold;
+  color: #333;
+}
+
+.skip-btn {
+  margin-top: 20px;
+  padding: 10px 20px;
+  font-size: 1em;
+  border: 1px solid #000;
+  background-color: transparent;
+  cursor: pointer;
+  transition: 0.3s;
+}
+
+.skip-btn:hover {
+  background-color: #f0f0f0;
+}
+
+.signup-container {
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.signup-box {
+  padding: 20px;
+  max-width: 400px;
+  width: 100%;
+  background-color: #f5f5f5;
+  border-radius: 8px;
+  box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1);
+  text-align: center;
+}
+
+.signup-box h2 {
+  font-size: 2em;
+  font-weight: bold;
+}
+
+.signup-box p {
+  color: #666;
+  margin: 5px 0 20px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+}
+
+input[type="text"],
+input[type="password"] {
+  padding: 12px;
+  margin: 10px 0;
+  font-size: 1em;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  outline: none;
+  transition: 0.3s;
+}
+
+input[type="text"]:focus,
+input[type="password"]:focus {
+  border-color: #0077ff;
+}
```

Comment:
Move all these tag styles (line 64 to line 83) inside the `.signup-page` class (it's at the top of this class). Otherwise, this will change the style of all `input` elements across the entire web app